packages/vx-shape/src/shapes/Pie.tsx

```diff
@@ -29,28 +22,28 @@ export type PieProps<Datum> = {
   left?: number;
   /** Array of data to generate a Pie for. */
   data?: Datum[];
-  /** Invoked for each datum, returns the value for a given Pie segment/arc datum. */
-  pieValue?: NumAccessor<Datum>;
-  /** Comparator function to sort *arcs*, overridden by pieSortValues if defined. If pieSort and pieSortValues are null, arcs match input data order. */
-  pieSort?: null | ((a: Datum, b: Datum) => number);
-  /** Comparator function to sort arc *values*, overrides pieSort if defined. If pieSort and pieSortValues are null, arcs match input data order. */
-  pieSortValues?: null | ((a: number, b: number) => number);
   /** Optional render function invoked for each Datum to render something (e.g., a Label) at each pie centroid. */
   centroid?: (xyCoords: [number, number], arc: PieArcDatum<Datum>) => React.ReactNode;
   /** Inner radius of the Arc shape. */
-  innerRadius?: NumAccessor<Datum> | number;
-  /** Inner radius of the Arc shape. */
-  outerRadius?: NumAccessor<Datum> | number;
-  /** Inner radius of the Arc shape. */
-  cornerRadius?: NumAccessor<Datum> | number;
+  innerRadius?: ArcPathConfig<PieArcDatum<Datum>>['innerRadius'];
```

Comment:
any reason not to `Pick` these from `ArcPathConfig` and `PiePathConfig` like in other files? looks like the type annotations should match, though I guess some of them require re-names 😱

Reply:
The pie ones have rename. I will change this to Pick from the config.